puts/harness/src/lib.rs

```diff
@@ -0,0 +1,113 @@
+mod bindings {
+    #![allow(non_upper_case_globals)]
+    #![allow(non_camel_case_types)]
+    #![allow(non_snake_case)]
+    #![allow(improper_ctypes)]
+    #![allow(dead_code)]
+    #![allow(clippy::all)]
```

Comment:
```suggestion
    #![allow(warnings, unused)]
```

Reply:
I think I want to keep the specific whitelist in that case. Because we have full control on the code from which the bindings are generated, I would like to notice if some new warnings are generated.